runtime/taskcontext.go

```diff
@@ -63,12 +73,14 @@ type TaskInfo struct {
 // properties, and abortion notifications.
 type TaskContext struct {
 	TaskInfo
-	webHookSet *webhookserver.WebHookSet
-	logStream  *stream.Stream
-	mu         sync.RWMutex
-	queue      client.Queue
-	status     TaskStatus
-	cancelled  bool
+	webHookSet  *webhookserver.WebHookSet
+	logStream   *stream.Stream
+	logLocation string
```

Comment:
Hopefully Wander can add some documentation here, but logLocation is a string that's a path to where you can find the task log file, so absolute path.  This is within the task context, which means the separators are whatever are used on the platform the worker is running on.  In this case task context logLocation is the location of the temp log file that's passed into NewTaskContext which then should probably have similar documentation as to what that should be.  The value that's passed into NewTaskContext is supplied by the environment (outside of the engine) and I believe uses the os golang package for creating temporary locations and constructing paths with the appropriate path separator.